language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/util/references/ReferencesUtil.java

```diff
@@ -81,9 +81,10 @@ private ReferencesUtil() {
         }
         
         Symbol symbol = symbolAtCursor.get();
-        project.get().currentPackage().modules().forEach(module -> {
-            List<Location> references = module.getCompilation().getSemanticModel().references(symbol);
-            moduleLocationMap.put(module, references);
+        project.get().currentPackage().moduleIds().forEach(moduleId -> {
+            List<Location> references = project.get().currentPackage()
+                    .getCompilation().getSemanticModel(moduleId).references(symbol);
+            moduleLocationMap.put(project.get().currentPackage().module(moduleId), references);
```

Comment:
Get the module to a variable, in order to improve the readability here

Reply:
Resolved in 1907cf8